assembly/massa-sc-runtime/unsupported_builtin.ts

```diff
@@ -3,4 +3,9 @@ export function main(): void {
     // Note: hrtime obtains the system's monotonic high resolution time
     //       this is not compatible with deterministic execution so we do not support it
     let hrt: u64 = process.hrtime();
+
+
+    // getRandomValues are not supported because it is not determinist
+    const array = new Uint8Array(10);
```

Comment:
The test will fail on line 5 (calling process.hrtime) so here what you need to do is define 2 functions and in the unit test call these 2 functions (and check it will fail)